src/loaders/anndata-loaders/AnnDataLoader.js

```diff
@@ -0,0 +1,11 @@
+import AbstractTwoStepLoader from '../AbstractTwoStepLoader';
+
+export default class AnnDataLoader extends AbstractTwoStepLoader {
+  loadCellNames() {
+    return this.dataSource.loadObsIndex();
+  }
+
+  loadGeneNames() {
+    return this.dataSource.loadVarIndex();
+  }
```

Comment:
Just a thought. I like the idea, but if these are just aliases to methods defined on `dataSource` (which is already the same for all anndata-derived loaders), then we might over-engineering the solution.

I see the benefit of having a clearer name in the loader context, but why not just name it that way on the `AnnDataSource` (i.e. `dataSource.loadGeneNames`), or just use a good variable name when calling this method on the loader:

```javascript
/* ... */
const geneNames = this.dataSource.loadVarIndex();
```

A tradeoff here is that it hides in the derived classes that `this.loadCellNames` is actually calling into the `dataSource`. Another thought is that looking at the function definition for `this.loadGeneNames` is really two navigation away instead of one.